cfgov/jinja2/v1/_includes/atoms/input.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   form.input()
+
+   ==========================================================================
+
+   Description: Builds form text field.
+
+   label:       Name of the field.
+
+   type:        Type of field. Will default to text, but can change to other
+                types to take advantage of HTML5 validation.
+                Possible fields: text, tel, email, url, number, date.
+
+   size:        The size of the field. Defaults to 1 (full-width).
+                Possible options: 1, 1-2, 1-4
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update({
+        'input': {
+            'label':        'This is a label',
+            'type':         'text',
+            'helperText':   lipsumText,
+            'placeholder':  'mm/dd/yyy',
+            'size':         '1-2',
+            'required':     true,
+            'disabled':     false
+        }
+    }) %}
+    {% set value = global_dict.input %}
+{% endif %}
+
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set id_slug = slugify( value.label ) %}
+<div class="form-l_col form-l_col-{{ value.size if value.size else '1'}}">
+    <label class="form-label-header" for="form_{{ id_slug }}">
+      {{ value.label }}
+      {%- if value.required == false -%}
+          <span class="micro-copy">&nbsp;(Optional)</span>
+      {%- endif -%}
+    </label>
+    {%- if value.helperText -%}
+        <p class="micro-copy" id="helpertext_{{id_slug}}">{{ value.helperText | safe }}</p>
```

Comment:
If two inputs had the same label, it would also break the labels. We really need to come up with a way to programatically generate them to be unique. I've done that by slugifying the labels, but we might we might want to add another technique.

Helper Text is used for to give additional helper text to screenreader outside of the label.